eregs.py

```diff
@@ -21,7 +23,7 @@ def cli(debug):
     if debug:
         log_level = logging.DEBUG
         sys.excepthook = lambda t, v, tb: ipdb.post_mortem(tb)
-    coloredlogs.install(level=log_level, fmt="%(levelname)s %(message)s")
+    coloredlogs.install(level=log_level)
```

Comment:
Let's try to retrieve that from the env then, and fall back to that as a default.